src/core/main.cpp

```diff
@@ -137,6 +142,14 @@ int main( int argc, char * * argv )
 		}
 	}
 
+#ifndef LMMS_WIN32
```

Comment:
Yeah, @Lukas-W should probably look into why it is failing... Looks like Homebrew is puking on a package dependency.  I can see why it would be particularly annoying for those that have many open pull requests.... :)